src/Microsoft.Health.Dicom.Core/Features/Validation/StringElementValidation.cs

```diff
@@ -22,10 +22,25 @@ public void Validate(DicomElement dicomElement, ValidationLevel validationLevel
         if (!string.IsNullOrEmpty(value) && validationLevel == ValidationLevel.Default)
             value = value.TrimEnd('\0');
 
-        ValidateStringElement(name, value, dicomElement.ValueRepresentation, dicomElement.Buffer);
+        if (IsNullOrEmpty(value))
+        {
+            // By default we will allow null or empty string and not go further with validation
+            return;
+        }
+
+        ValidateStringElement(name, dicomElement.ValueRepresentation, value, dicomElement.Buffer);
     }
 
-    protected abstract void ValidateStringElement(string name, string value, DicomVR vr, IByteBuffer buffer);
+    protected abstract void ValidateStringElement(string name, DicomVR vr, string value, IByteBuffer buffer);
+
+    protected virtual bool GetValue(DicomElement dicomElement, out string value)
+    {
+        value = dicomElement.GetFirstValueOrDefault<string>();
+        return string.IsNullOrEmpty(value);
+    }
 
-    protected abstract bool GetValue(DicomElement dicomElement, out string value);
+    protected virtual bool IsNullOrEmpty(string value)
```

Comment:
This should just be a property at the top like `AllowNullOrEmpty` that can be overridden:

```csharp
protected virtual bool AllowNullOrEmpty => false;
```

Then you can check in `Validate`:
```csharp
if (!AllowNullOrEmpty && string.IsNullOrEmpty(value))
    return;
```